### PR TITLE
Update WasabiSetupVM.md

### PR DESCRIPTION
**Pull request by sigit-io**
Some documentation suggestions following a (successful) Qubes install

### CODE REVIEW

**Review by yahiheb — APPROVED** [review]
LGTM

**Review by MaxHillebrand — DISMISSED** [review]
Thanks for the contribution!

**Review by MarnixCroes — COMMENTED** [review]
thank you for taking the time to update this

File: docs/using-wasabi/WasabiSetupVM.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ You can disable .NET's telemetry, which is sending some usage information to Mic
 ```sh
 [user@template-wasabi ~]$ export DOTNET_CLI_TELEMETRY_OPTOUT=1
 ```
+To make this persistent, add `DOTNET_CLI_TELEMETRY_OPTOUT=1` to the `/etc/environment` file.
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupVM.md:76]:
```suggestion

To make this persistent, add `DOTNET_CLI_TELEMETRY_OPTOUT=1` to the `/etc/environment` file. 
```

File: docs/using-wasabi/WasabiSetupVM.md [hunk 2]
```diff
@@ -94,6 +95,8 @@ All these App VMs will be updated when you update `template-wasabi`.
 [user@dom0 ~]$ qvm-create source-wasabi --template template-wasabi --label red
 ```
 
+Ensure the VM has at least 5GB of disk available for the upcoming build (default is 2gb)
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupVM.md:98]:
```suggestion
Ensure the VM has at least 5GB of disk available for the upcoming build (default is 2gb).
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]

**Review by MarnixCroes — APPROVED** [review]
ACK

**Comment by MarnixCroes** [comment]
thanks @sigit-io

**Pull request merged**